templates/experiments/components/experiment_actions_column.html

```diff
@@ -1,7 +1,7 @@
-<form method="post" action="{% url 'experiments:start_authed_web_session' record.team.slug record.id %}" class="inline">
+<form method="post" action="{% url 'experiments:start_authed_web_session' record.team.slug record.id record.get_working_version.version_number %}" class="inline">
```

Comment:
This doesn't seem necessary since this table only shows the working versions. But also doesn't harm.